packages/flutter/lib/src/material/expand_icon.dart

```diff
@@ -179,7 +185,7 @@ class _ExpandIconState extends State<ExpandIcon> with SingleTickerProviderStateM
         onPressed: widget.onPressed == null ? null : _handlePressed,
         icon: RotationTransition(
           turns: _iconTurns,
-          child: const Icon(Icons.expand_more),
+          child: const Icon(iconData ?? Icons.expand_more),
```

Comment:
```suggestion
          child: iconData ? const Icon(iconData) : const Icon(Icons.expand_more),
```

May be able to put a single const before after `child:`

Reply:
Thanks for the review!

I am confused about this.
Isn't
```dart
child: const Icon(iconData ?? Icons.expand_more),
```
shorter than your suggestion?